### PR TITLE
[I18N] project,_timesheet_holidays: fix typo's / improve terms

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: resolve two typo's.

Current behavior before PR: 'Gahtering' instead of 'Gathering'

Desired behavior after PR is merged: 'Gathering' instead of 'Gahtering'.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@mart-e for you.

**Comment by Yenthe666** [comment]
@mart-e combined and improved as asked ;)

**Comment by Yenthe666** [comment]
@mart-e let me know if you need anything else.

File: addons/project_timesheet_holidays/models/hr_holidays.py [hunk 1]
```diff
@@ -45,9 +45,9 @@ def _check_timesheet_generate(self):
         for holiday_status in self:
             if holiday_status.timesheet_generate:
                 if not holiday_status.timesheet_project_id or not holiday_status.timesheet_task_id:
-                    raise ValidationError(_('Both the internal project and task are required to\
-                    generate timesheet for the time of. If you don\'t want timesheet, you must let\
-                    empty internal project and task.'))
+                    raise ValidationError(_("Both the internal project and task are required to "
+                    "generate a timesheet for the time off. If you don\'t want a timesheet, you should "
```
Comment (mart-e) [inline addons/project_timesheet_holidays/models/hr_holidays.py]: no need for escaping the `\` if you are using `"1

**Comment by mart-e** [comment]
Thanks for the merge

@robodoo r+

PS: "internationalization" tag is about translations. Rephrasing text is not a translation issue, it is a content issue ;-)

**Comment by Yenthe666** [comment]
Thanks for the merge and noted! ;-)

**Pull request closed without merging**